pwa/src/components/logs/logTable/logTable.tsx

```diff
@@ -11,7 +11,7 @@ interface LogsTableProps {
   logs: any;
 }
 
-const LogsTable: React.FC<LogsTableProps> = ({ logs }) => {
+const LogsTable: React.FC<LogsTableProps> = ({logs}) => {
```

Comment:
This component still has a lot of errors. You can call me and we can then solve them together.